### DISCUSSION TITLE
android:banner configuration. either in expo or in expo-build-properties

### DISCUSSION THREAD

**Discussion by silencer07** [expo/expo#27142] [Proposals and Ideas]
seems that there is a wiki available on how to make an expo app work in android tv, but the android:banner configuration is missing

https://gist.github.com/miazga/2e6449e0c591e3ac8e22185b2edb447d

Any clue how to make it work aside from making a manual edit?

**Comment by douglowder** [comment]
The latest version of `@react-native-tvos/config-tv` (0.0.5) supports adding an Android TV banner image. See this example: https://github.com/react-native-tvos/IgniteTV/commit/0a29a6de7aa742acc4e9a87b7f358c2e3be2fafb

You just set that property, add the image to your repo, and rerun `npx expo prebuild --clean`.

**Reply by michalpospiech** [reply]
Thanks a lot for this update!

**Reply by silencer07** [reply]
@douglowder i tried using the plugin, but seems that I cannot make it work. I deleted the android folder as well

EDIT: I downloaded the igniteTV sample repo and did `npx expo prebuild` and I am not seeing the `android:banner` as well?
